src/transformers/models/dpt/configuration_dpt.py

```diff
@@ -76,6 +79,8 @@ class DPTConfig(PretrainedConfig):
             - "project" passes information to the other tokens by concatenating the readout to all other tokens before
               projecting the
             representation to the original feature dimension D using a linear layer followed by a GELU non-linearity.
+        embedding_type (`str`, *optional*, defaults to `"patch_embedding"`):
+            The type of embedding to use. Can be one of [`"patch_embedding"`, `"vit_hybrid"`].
```

Comment:
(nit) I think "hybrid" is a better name for the embedding_type, as "vit_hybrid" is an entire backbone, not really an embedding type.

Reply:
Fixed in 03eac8f - aso updated the model config on the Hub